Server/src/loaders/loadStartupProcess.ts

```diff
@@ -5,6 +5,7 @@ import { connectDB } from '../database'
 import cors from 'cors';
 import express from 'express';
 import { fileUploadRouter } from '../routes/fileUploadRouter'
+import { getDataRouter } from '../routes/queryDataRouter';
```

Comment:
I think we should try to be careful with what we consider to be refactoring... Ideally its when we restructure our codebase to make it more optimal and/or more flexible

Reply:
You're right, I was misusing the term here. It won't happen in the future.